ERCS/erc-7596.md

```diff
@@ -0,0 +1,2756 @@
+---
+eip: 7596
+title: Equity Shares Book-Entry System
+description: An auto-pilot book-entry system of equity shares to form web3.0 capital markets by implementing concept of "Code is Law".
+author: Paul Lee (@paul-lee-attorney) <paul_lee0919@163.com>
+discussions-to: https://ethereum-magicians.org/t/add-erc-equity-shares-book-entry-system/18155
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-15
+---
+
+## Abstract
+
+This ERC describes the details of the Equity Shares Book-Entry System (the "**System**"), a proposed protocol and ABI definition that can automatically manage the book-entry records of a company with respect to its equity shares and corporate governance behaviors, in accordance with the consitutional document thereof, so as to enable investors, shareholders, managers, and other stakeholders can directly engage in almost all equity share transactions and corporate governance activities under the automatic control of smart contracts, including share subscription, share transfer, share pledge, share consideration payment, share listing, constitution updates, contracts signing, motion submission, voting, candidates nomination, taking office and resignation etc..
```

Comment:
This is an extremely long sentence. While not strictly required, I'd very much recommend breaking it up to make reading easier :rofl:

Reply:
Thank you for your suggestion. I will try to simplify the wording with plain language.